ActionSheetPicker-3.0.xcworkspace/xcshareddata/ActionSheetPicker-3.0.xccheckout

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
3) please, remove this file from git, (and add it to gitignore ) since it shouldn't be in repo.